### DISCUSSION TITLE
🧋Welcome to Bubble Tea

### DISCUSSION THREAD

**Discussion by meowgorithm** [charmbracelet/bubbletea#71] [General]
## Hello!

This is the place to discuss Bubble Tea stuff. Ask questions, share your stuff, and say hello.

**Comment by ar1ja** [comment]
but i am Bubble Tea

**Reply by ar1ja** [reply]
Sad, this joke makes no sense anymore, I used to be named B00bleaTeA

**Reply by meowgorithm** [reply]
Oh I remember

**Comment by jonas-kgomo** [comment]
how do we simply install this on a Mac?

**Reply by muesli** [reply]
If you just want to play with the demos / examples, you can simply clone this repository, `cd` to an example dir and run `go build`. This will produce a binary that you can then execute.
